src/portal/lib/src/tag/tag.component.html

```diff
@@ -75,7 +75,7 @@ <h3 class="modal-title">{{ 'REPOSITORY.RETAG' | translate }}</h3>
               </div>
             </clr-dropdown-menu>
           </clr-dropdown>
-          <button type="button" class="btn btn-sm btn-secondary" [disabled]="!(selectedRow.length===1 && guestRoleOrAbove)" (click)="retag(selectedRow)"><clr-icon shape="copy" size="16"></clr-icon>&nbsp;{{'REPOSITORY.RETAG' | translate}}</button>
+          <button type="button" class="btn btn-sm btn-secondary" [disabled]="!(selectedRow.length===1 && guestRoleOrAbove) || withAdmiral" (click)="retag(selectedRow)"><clr-icon shape="copy" size="16"></clr-icon>&nbsp;{{'REPOSITORY.RETAG' | translate}}</button>
```

Comment:
This just disabled the button, I think what we want to do is 'hide' the button if I'm not mistaken.
The button should not be displayed.